emscripten/ARToolKitJS.cpp

```diff
@@ -160,6 +170,9 @@ extern "C" {
 			if (arFilterTransMat(arc->ftmi, transF, reset) < 0) {
 					ARLOGe("arFilterTransMat error with marker %d.\n", markerIndex);
 			}
+
+			matrixLerp(transF, transFLerp, 24.0);
```

Comment:
try with another interpolationFactor, maybe 12.0 instead of 24.0?

Reply:
yes we have to try different values